src/coreclr/tools/aot/ilc.sln

```diff
@@ -18,9 +18,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ILCompiler.LLVM", "ILCompil
 EndProject
```

Comment:
Solution changes - to be reverted?

Reply:
yes, I don't know why VS does this all the time for me.  I'll clear my .vs folder.

Reply:
I see, VS wants to create the configuration entries for the 2 vcxproj jit entries which are missing.  Does that not happen for you @SingleAccretion ?

Reply:
It certainly does add things, but I always `git restore` + `Ignore` them (so not sure what they actually are).

Would be no objections from me for checking it in and seeing whether VS holds its peace from there.